jobs/JJOB_PREP_LETKF

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-#set -e
+#set -x
```

Comment:
Why is -e removed?

Reply:
Not intentionally, probably during automerging.

Reply:
I updated this and it was failing because of the soft links of ice and ocean.pert.*.nc, these soft links have been removed, and they are generated by hofx of each ensemble member.